### ISSUE TITLE
Testsuite subhalive hangs on d2b91df8182d23fe3d0b7b1b2f885db07b7a6e06

### ISSUE THREAD

**Issue by deepfire** [lukexi/halive#18]
```
Running 2 test suites...
Test suite demo: RUNNING...
MAIN BEGIN
demo: SDLCallFailed {sdlExceptionCaller = "SDL.Init.init", sdlFunction = "SDL_Init", sdlExceptionError = "No available video device"}
Test suite demo: FAIL
Test suite logged to: dist/test/halive-0.1.0.8-demo.log
Test suite subhalive: RUNNING...
```
Commit-id: d2b91df8182d23fe3d0b7b1b2f885db07b7a6e06

**Comment by lukexi** [comment]
Hm, it runs fine for me on Windows and Mac, are you on Linux? That test is for an OpenGL/SDL2 app and I haven't tested it there. If you can get https://hackage.haskell.org/package/sdl2 to work then that test should work too.

You can run `halive Computation.hs` (and then edit and save `Computation.hs`) to check if things are working for terminal-only usage.

I have the tests in the cabal file mainly to make installing their dependencies easier; they're really more of a couple of quick test programs than an actual test suite of any kind : ).

**Comment by deepfire** [comment]
@lukexi, yes, this is on Linux.  The testsuite fails, hackage-wide automated testing setups get upset.. : -)

**Comment by deepfire** [comment]
Not a problem per se, a mere inconvenience.  Will you recommend to disable tests by default?
